R/load_fonts.R

```diff
@@ -8,24 +7,34 @@
 #' @export
 #'
 load_arcadia_fonts <- function(custom_font = "Suisse", fallback_font = "sans") {
-  # define font names to check for
-  font_names <- c("Suisse Int'l", "Suisse Int'l Semi Bold", "Suisse Int'l Medium", "Suisse Int'l Mono")
+  # check if custom font available
+  font_family <- custom_font
```

Comment:
Yeah you're right, the renaming is confusing, probably because the comments are out of sync with the code now. Let me refactor this so it's clearer that we are defining the font to import as either `custom_font` or `fallback_font`.